subsys/usb/device_next/class/bt_hci.c

```diff
@@ -35,6 +35,113 @@
 #include <zephyr/drivers/bluetooth.h>
 #include <zephyr/sys/atomic.h>
 
+// FIXME: HACK
```

Comment:
NACK... please no specific Controller implementation to be referenced in the USB subsystem.

Reply:
This is a hack I use while I work on other parts. Have you read my idea to get notion of which handles are ISO by capturing HCI commands? Do you think there might be a better idea?